libcontainer/cgroups/utils.go

```diff
@@ -223,6 +231,29 @@ func GetInitCgroupDir(subsystem string) (string, error) {
 	return getControllerPath(subsystem, cgroups)
 }
 
+func GetInitCgroupPath(subsystem string) (string, error) {
```

Comment:
We don't need it now, but the systemd cgroup manager does use `GetInitCgroup`. I can drop it if you prefer, this is more for the benefit of users of `libcontainer`.